### PR TITLE
lca ibi: Added WithCABundle function.

### PR DESCRIPTION
**Pull request by achuzhoy**
Added the ability to set ici.spec.caBundleRef field.

### CODE REVIEW

**Comment by achuzhoy** [comment]
cc @trewest @kononovn

File: pkg/ibi/imageclusterinstall.go [hunk 1]
```diff
@@ -196,6 +196,25 @@ func (builder *ImageClusterInstallBuilder) WithExtraManifests(extraManifestsName
 	return builder
 }
 
+// WithCABundle sets a CA bundle via configmap name.
+func (builder *ImageClusterInstallBuilder) WithCABundle(caBundleConfigMapName string) *ImageClusterInstallBuilder {
+	if valid, _ := builder.validate(); !valid {
+		return nil
```
Comment (trewest) [inline pkg/ibi/imageclusterinstall.go:202]:
```suggestion
		return builder
```

Comment (achuzhoy) [reply pkg/ibi/imageclusterinstall.go]: addressed thanks.

**Review by trewest — APPROVED** [review]
/lgtm

**Pull request merged**